examples/kubernetes/clustertemplates/main.tf

```diff
@@ -0,0 +1,5 @@
+data "vkcs_kubernetes_clustertemplates" "templates" {}
+
+output "available_templates_by_name" {
+  value = [ for template in data.vkcs_kubernetes_clustertemplates.templates.cluster_templates: {name = template.name, version = template.version}]
```

Comment:
Have you investigated the output? So if we need to separate examples for searching by version and name?

Reply:
one version one name it was. everytime

Reply:
So if we need to separate examples for searching by version and name?

Reply:
we will use mostly only version because it doesnt contain minor version and name looks too long:
{
    "name" = "Kubernetes-almalinux-v1.24.15-mcs.1"
    "version" = "1.24"
  },